Assets/Scripts/Player/PlayerShooting.cs

```diff
@@ -11,18 +11,37 @@ public sealed class PlayerShooting : MonoBehaviour
 
     private GameObject _bullet;
     private float _shootingPower;
+    private float _shootingDelay;
+    private float _currentShootDelay;
+    private bool _autoShooting;
 
-    private void Awake() => SetGunData(startingGunData);
+    private void Awake()
+    {
+        SetGunData(startingGunData);
+        _currentShootDelay = startingGunData.shootingDelay;
```

Comment:
Waarom is dit niet gewoon onderdeel van `SetGunData()`?

Reply:
![image](https://github.com/Team-Swamp/Brackeys-Gamejam-2023.2/assets/70578065/a3cf723f-19fe-4355-bc39-3b1f99bed3dc)

Ja nee, code zegt kan niet.